app.py

```diff
@@ -76,5 +76,27 @@ def skill():
 
     if request.method == 'POST':
         return jsonify({})
+        
 
     return jsonify({})
+
+
+@app.route('/resume/skill/<int:index>', methods=['GET'])
+def get_skill_by_index(index):
+    """
+    Get a specific skill by index
+    """
+    try:
+        skill_index = data['skill'][index]
+        return jsonify(skill_index)
+    except IndexError:
+        return jsonify({"error": "Skill not found"})
+    
+
+@app.route('resume/skill/<int:index>', methods=['DELETE'])
+def delete_skill(index):
+    if 0 <= index < len(data["skill"]):
+        data["skill"].pop(index)
+        return jsonify({"deleted": "Successfully deleted skill"})
```

Comment:
To keep the messages consistent, let's return {"message": "Successfully deleted skill"}. Also, let's return 200 code as well